packages/language-server/src/server.ts

```diff
@@ -174,18 +178,29 @@ documents.onDidChangeContent(async (changeEvent) => {
   }
 });
 
-connection.onCodeAction((params) => {
+connection.onCodeAction(async (params) => {
+  if (semanticModelLoaded === undefined) {
+    return;
+  }
+
+  const ui5Model = await semanticModelLoaded;
   const docUri = params.textDocument.uri;
   const textDocument = documents.get(docUri);
   if (textDocument === undefined) {
     return undefined;
   }
 
   const diagnostics = params.context.diagnostics;
-  const codeActions = compact(
-    map(diagnostics, (_) => diagnosticToCodeActionFix(textDocument, _))
-  );
+  const nonStableIdIssue = find(diagnostics, (_) => _.code === 1000);
```

Comment:
feels like this condition is too specific for stableID fixes.
Should/could we preform this check inside `diagnosticToCodeActionFix`?